likecoin/js/admin/likecoin_metabox.js

```diff
@@ -1,9 +1,73 @@
-/* global jQuery, wpApiSettings, lcPostInfo */
+/* global jQuery, wpApiSettings, lcPostInfo, lcStringInfo */
+
+const mainTitleField = document.querySelector('#lcTitleStatus');
+const ISCNStatusTextField = document.querySelector('#lcISCNStatus');
+function createElementWithAttrbutes(el, {
+  text, className, id, rel, target, href,
+}) {
+  const element = document.createElement(el);
+  if (text) element.innerText = text;
+  if (id) element.setAttribute('id', id);
+  if (className) element.setAttribute('class', className);
+  if (rel) element.setAttribute('rel', rel);
+  if (target) element.setAttribute('target', target);
+  if (href) element.setAttribute('href', href);
+  return element;
+}
+function refreshMainTitleField(signalCSSClass, text) {
+  mainTitleField.textContent = '';
+  const statusDot = createElementWithAttrbutes('h1', {
+    text: ' · ',
+    className: signalCSSClass,
+  });
+  const statusText = createElementWithAttrbutes('h3', {
+    text,
+    className: 'iscn-status-text',
+  });
+  mainTitleField.appendChild(statusDot);
+  mainTitleField.appendChild(statusText);
+}
+function generateMainStatusText(status) {
+  const {
+    mainStatusLoading,
+    mainStatusLIKEPay,
+    mainStatusUploadArweave,
+    mainStatusRegisterISCN,
+  } = lcStringInfo;
+  let mainStatusText;
+  switch (status) {
+    case 'loading':
+      mainStatusText = mainStatusLoading;
+      break;
+    case 'onLIKEPay':
+      mainStatusText = mainStatusLIKEPay;
+      break;
+    case 'onUploadArweave':
+      mainStatusText = mainStatusUploadArweave;
+      break;
+    case 'onRegisterISCN':
+      mainStatusText = mainStatusRegisterISCN;
+      break;
+    default:
+      mainStatusText = '-';
+  }
+  return mainStatusText;
+}
+function refreshMainStatusField(status) {
```

Comment:
`updateMainStatusField` is a better name

Reply:
Since all fields use this technique to replace children, how about make `ISCNStatusTextField` a input param and use this function for all status fields?